tests/lib/websocket/test_interest_type.py

```diff
@@ -0,0 +1,26 @@
+from unittest.mock import patch
+
+from apollo.lib.websocket.interest_type import (InterestTypeFunctionHandler,
+                                                WebSocketObserverInterestType)
+from apollo.models.agent import Agent
+
+
+def test_interest_type_function_handler_run_corresponding_function(db_session):
+    agent = Agent(name='test')
+    db_session.add(agent)
+    db_session.commit()
+
+    data = InterestTypeFunctionHandler().run_corresponding_function(
+        WebSocketObserverInterestType.AGENT_LISTING
+    )
+    assert len(data) == 1
+    assert data[0]['connection_state'] == 'disconnected'
```

Comment:
```
assert data[0] == {
	'id': agent_id,
	'name': 'test',
	'connection_state': 'disconnected'
}
```

May as well be complete in your assertion. Asserting one value seems a bit random.